kubernetes/helm/collabora-online/values.yaml

```diff
@@ -1,10 +1,10 @@
-# Default values for libreoffice-online.
+# Default values for collabora-online.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 global:
   app:
-    name: libreoffice-online
-    version: 6.5.0
+    name: collabora-online
+    version: 6.4.4
```

Comment:
Do we have to change version number here always, when we bump version of the release?

Reply:
I don't think so, I will test the setup without ^this version specification.